### PR TITLE
bunnyfetch: init at unstable-2021-05-24

### PR DESCRIPTION
**Pull request by devins2518**
###### Motivation for this change
bunnyfetch: init at 0.2

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Added a release notes entry if the change is major or breaking
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -2415,6 +2415,12 @@
     githubId = 896182;
     name = "devhell";
   };
+  devins2518 = {
```
[the remark this answers is not in the document]
Comment (devins2518) [reply maintainers/maintainer-list.nix:2418]: fixed in d5a1bb6

File: pkgs/tools/misc/bunnyfetch/default.nix [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "bunnyfetch";
+  version = "0.2";
```
[the remark this answers is not in the document]
Comment (devins2518) [reply pkgs/tools/misc/bunnyfetch/default.nix]: fixed in fe753da

File: pkgs/tools/misc/bunnyfetch/default.nix [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+  src = fetchFromGitHub {
+    owner = "Mewyuna";
+    repo = pname;
+    rev = "7bcc45fb590b37f410e60af893e679eb0729ecb1";
+    sha256 = "1lgqrwmxdxd1d99rr0akydfwcsbcmz75fkbq9zrl842rksnp5q3r";
+  };
+
+  vendorSha256 = "1vv69y0x06kn99lw995sbkb7vgd0yb18flkr2ml8ss7q2yvz37vi";
+
+  doCheck = false;
```
Comment (lukegb) [inline pkgs/tools/misc/bunnyfetch/default.nix:16]: Can you leave a comment about why tests are disabled? (e.g. no tests in upstream)

Comment (devins2518) [reply pkgs/tools/misc/bunnyfetch/default.nix:17]: Added in 0885a2d.

**Pull request merged**